invokeai/app/api/routers/sessions.py

```diff
@@ -38,6 +35,99 @@ async def create_session(
     return session
 
 
+@session_router.post(
```

Comment:
Should this be on a new batches_router? Seems weird to have URLs prefixed with session but NBD

Reply:
Agreed, I'll segregate those. Believe this is a holdover from MVP of the feature.